pedestrian/AgentsSourcesManager.cpp

```diff
@@ -214,7 +214,9 @@ void AgentsSourcesManager::ComputeBestPositionCompleteRandom(AgentsSource* src,
 {
      auto dist = src->GetStartDistribution();
      auto subroom = _building->GetRoom(dist->GetRoomId())->GetSubRoom(dist->GetSubroomID());
-     vector<Point> positions = PedDistributor::PossiblePositions(*subroom);
+     auto config=GetBuilding()->GetConfig();
+     auto distributor = std::unique_ptr<PedDistributor>(new PedDistributor(config));
+     vector<Point> positions = distributor->PossiblePositions(*subroom);
```

Comment:
Why was this changed?

Reply:
> Why was this changed?

Because I have changed the function PossiblePosition from static to non-static.
May be there are better way to doi it.